### ISSUE TITLE
Bulk add shows

### ISSUE THREAD

**Issue by 8cylinder** [8cylinder/tv-overlord#21]
Find a method to add bulk shows.

### Read from a csv file?
A csv file with headers, "show name", "season", "episode"
`tvol bulkadd shows.csv`

### Command line?  
Loop over some show information source:

``` bash
for show in $("show data source"); do
    showname=$("get show name from somewhere")
    season=$("get season from somewhere")
    episode=$("get episode from somewhere")
    tvol addbulk --show-name $showname --season $season --episode $episode
done
```

**Comment by funkyfuture** [comment]
what would that do? mark as watched?

if so, i think some flexibility would be neat:

```
Dr. Who
Dr. Who,1
Dr. Who,1,-7
```

**Comment by jonnagel** [comment]
@8cylinder 
I like your idea, it's much better than my original thought.

I was just thinking it would be nice to have a command that sets all the last value in `tvol list`

My simple solution to add shows was to manually script it in bash.

**Comment by 8cylinder** [comment]
Fixed in dev branch: 1b850bae7de0d287f853c8852553f705786a5ecc, e8d48e3070b2bb2606a45cdd4b32bb7cf9997bbc
